src/mnemonic.rs

```diff
@@ -0,0 +1,139 @@
+use std::fmt;
+use std::ops::*;
+use std::marker::PhantomData;
+use rand::{CryptoRng, Rng};
+
+use horcrux::field::Field;
+use horcrux::gf2n::{GF128, GF256};
+use horcrux::shamir::{Shamir};
+
+#[cfg(feature = "bip39")]
+use bip39::{Mnemonic, Language};
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct Bip39<F> {
```

Comment:
I don't see the point of this struct, which is exactly encapsulating the underlying field without changing any behavior.